themes/bootstrap5/templates/RecordDriver/DefaultRecord/result-list.phtml

```diff
@@ -51,7 +51,7 @@
         <?php else: ?>
           <?php
             $journalTitle = $this->driver->getContainerTitle();
-            $summDate = $this->driver->getPublicationDates();
+            $summDate = $this->driver->getHumanReadablePublicationDates();
```

Comment:
Have you done any analysis to be sure this change won't have weird side effects outside of ProQuest? (I haven't looked closely, just double-checking that this is safe and intentional since it impacts a base template; happy to dig in deeper if you'd like me to invest the time).

Reply:
Yeah fair point, I did some quick display of Solr results and didn't see any difference, and I figured if there was going to be a difference, it would be a good once since this is a template and so the 'human readable' thing presumably applies.  But that's not proof, and if it's safer I can roll it back and adjust the new driver accordingly.

Reply:
I'll take a closer look at this tomorrow with a fresh brain and see how concerned I feel about it. ;-)

Reply:
I took another look, and I agree that this seems fine. Thanks!